testConf/conf/default.yaml

```diff
@@ -0,0 +1,15 @@
+api:
+  host: "127.0.0.1"
+  port: 8080
+
+foo_service:
+  foo_string: foo
+  foo_int: 42
+  foo_float: 9.81
+  foo_bool: false
+
+bar_service:
```

Comment:
Why did we change the structure?

Reply:
The structure is the same, this is merely an example.